### PR TITLE
nitrokey3: Add nitropy documentation

### PR DESCRIPTION
**Pull request by robin-nitrokey**
This patch adds documentation on using the Nitrokey 3 with nitropy.

Marked as a draft until the next pynitrokey release.

### CODE REVIEW

**Comment by robin-nitrokey** [comment]
pynitrokey 0.4.8 has been released with Nitrokey 3 support, so this should be ready to review and merge.

File: nitrokey3/shared/nitropy.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
+Using the Nitrokey 3 with nitropy
+---------------------------------
+
+.. contents:: :local:
+
+``nitropy`` can be used to list and mange Nitrokey 3 devices.
```
Comment (szszszsz) [inline nitrokey3/shared/nitropy.rst:6]: typo - mange

**Review by szszszsz — CHANGES_REQUESTED** [review]
Looks good. Please check the mentioned typo.

**Comment by robin-nitrokey** [comment]
Thanks, fixed!

**Pull request merged**